asistente_ladm_col/gui/create_boundaries_cadastre_wizard.py

```diff
@@ -84,23 +85,40 @@ def finished_dialog(self):
     def prepare_boundary_creation(self):
         # Load layers
         self._boundary_layer = self.qgis_utils.get_layer(self._db, BOUNDARY_TABLE, load=True)
+
         if self._boundary_layer is None:
             self.iface.messageBar().pushMessage("Asistente LADM_COL",
                 QCoreApplication.translate("CreateBoundariesCadastreWizard",
                                            "Boundary layer couldn't be found... {}").format(self._db.get_description()),
                 Qgis.Warning)
             return
 
+        self._boundary_point_layer = self.qgis_utils.get_layer(self._db, BOUNDARY_POINT_TABLE, load=True)
```

Comment:
Instead of calling several times `qgis_utils.get_layer()`, use `qgis_utils.get_layers()`, which is more efficient for multiple layers.

Reply:
Thanks, the changes are this [commit](https://github.com/AgenciaImplementacion/Asistente-LADM_COL/pull/139/commits/63b9eb061769f48a07781e0f913ff2a83f41b231)